packages/shared/src/graphql/fragments.ts

```diff
@@ -32,6 +32,46 @@ export const USER_SHORT_INFO_FRAGMENT = gql`
   }
 `;
 
+export const CONTENT_PREFERENCE_FRAMENT = gql`
+  fragment ContentPreferenceFragment on ContentPreference {
+    referenceId
+    user {
+      id
+      name
+      image
+      username
+    }
```

Comment:
I lowered amount of data we need here, I would even say that we might not even need anything, since `ContentPreference` is always on parent entity which will have its own id. Will log it so we can remove if not needed at the end of implementation at least.